automation/taskcluster/decision_task.py

```diff
@@ -168,13 +205,18 @@ def release(channel, is_staging, version_name):
         ordered_groups_of_tasks = raptor(result.staging)
     elif command == 'nightly':
         formatted_date = datetime.datetime.now().strftime('%y%V')
-        ordered_groups_of_tasks = release('nightly', result.staging, '1.0.{}'.format(formatted_date))
-    elif command == 'beta':
-        semver = re.compile(r'^v\d+\.\d+\.\d+-beta\.\d+$')
-        if not semver.match(result.tag):
-            raise ValueError('Github tag must be in beta semver format and prefixed with a "v", e.g.: "v1.0.0-beta.0"')
+        ordered_groups_of_tasks = nightly_to_production_app(result.staging, '1.0.{}'.format(formatted_date))
+    elif command == 'github-release':
         version = result.tag[1:]  # remove prefixed "v"
-        ordered_groups_of_tasks = release('beta', False, version)
+        beta_semver = re.compile(r'^v\d+\.\d+\.\d+-beta\.\d+$')
+        production_semver = re.compile(r'^v\d+\.\d+\.\d+(-rc\.\d+)?$')
+        if beta_semver.match(result.tag):
+            ordered_groups_of_tasks = release('beta', False, version)
```

Comment:
Nit: Should we pass `result.staging` instead of `False`? I see it wasn't the case before, but that would make staging releases easier, in my opinion.

Reply:
I would allow beta and production releases to be done in staging, at least for the sake of testing github-releases, which aren't covered by the nightly path. Let's address it, in case an external team asks for a staging release. I'm thinking of the QA team, for instance, which has made similar requests and were stuck because the code base doesn't have clean staging paths (for instance https://bugzilla.mozilla.org/show_bug.cgi?id=1553096#c10) 

It can be done in a follow up.

Reply:
Discuss offline: Okay to open staging releases up when a team asks for it. Chances are: we may regress something by the time a team requests it, so we'd still have some fixes to do anyway.